CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 UNRELEASED
 ----------
 
+* [ [#1772](https://github.com/digitalfabrik/integreat-cms/issues/1772) ] Fix hidden languages being listed in fallback translation text on the API endpoint
+
+
```

Comment:
```suggestion
```
Exactly two new lines between the last bullet point and the next version heading are required. Not sure why the check didn't complain, probably this is an open todo for #1689.